examples/dask/start-with-dask.ipynb

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
<p><code>dask_saturn.SaturnCluster()</code> will create a new cluster for you if you don't already have one connected to this resource. So I think that this line can / should be removed --&gt; "If you are having issues with this, you may not have a running Dask cluster on Saturn Cloud".</p><p> </p><p>Could you replace this language with something like the language used for this step in many of the other examples? See, for example, the "Initialize a Dask Cluster" section of https://github.com/saturncloud/examples/blob/47fcb28454a64b892574859c90377a21f49cbc94/examples/examples-cpu/nyc-taxi/xgboost-dask.ipynb.</p><p> </p><p>&lt;hr&gt;</p><p> </p><p>I also think there's a small grammatical error in the third sentence. I'd rewrite it as:</p><p> </p><blockquote>The code creates a function that computes exponents and runs it across a list of inputs in parallel. </blockquote>
<br/>

 _Reply via <a href='https://app.reviewnb.com/saturncloud/examples/pull/116/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
I didn't know it would create the cluster for you! Neat!

Reply:
Okay looking at the link you sent, I want to avoid going into too much detail about creating the cluster from within the notebook. I think it's confusing that the code creates the cluster once, and it's easier to think about having the cluster creation done though the UI, which I think we lead users towards in our design.

That said, I'll update the notebook to be more clear about exactly what it's doing in this case.